ecg_data.py

```diff
@@ -1,12 +1,100 @@
-class hrm:
+import numpy
+import pandas
+
+
+class ecg_data:
     'This is a class'
-    def __init__(self, filename, avgT1,avgT2,bradyT,bradyThresh,tachyT,tachyThresh):
-        self.avgT1 = 0.0
-        self.avgT2 = 10.0
 
-    #def instantHr(self):
+    def __init__(
+            self,
+            filename,
+            begin_time=0,
+            end_time=10,
+            bradyT=5,
+            bradyThresh=60,
+            tachyT=5,
+            tachyThresh=100):
+
+        # put peak detection here
+        data = pandas.read_csv(
+            filename, converters={
+                "times": float, "voltages": float})
+        voltages = data.voltages.values
+        finalTimes = [0]
+        avgvoltage = numpy.average(voltages)
+
+        threshvoltage = avgvoltage * 2
+
+        peaks = numpy.where(voltages >= threshvoltage)
+        peaks1 = peaks[0]
+        peaks2 = peaks1.tolist()
+        boxcar = []
+        for i in range(1, len(peaks2) - 1):
```

Comment:
The logic of what is going on here is hard to follow!  Would try to create some new methods that are descriptive of what they are doing.